app/javascript/src/components/Dashboard/Notes/HeaderMenu.jsx

```diff
@@ -0,0 +1,75 @@
+// @ts-nocheck
+import React, { useState } from "react";
+
+// @ts-ignore
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui";
+import { MenuBar } from "@bigbinary/neetoui/layouts";
```

Comment:
I want you to be careful here. I had explicitly pointed out how to handle this piece of code in the Slack channel, and thus I expect you to take care of it. If you are not taking care of it, then there are two possibilities:

- You don't agree with my suggestion or my suggestion doesn't work. Which I am fine with. We can discuss about it and rectify.
- You are still not attending to all Slack notifications. For this we will need to have a more serious talk.

Please do point out what exactly is the reason why you have not applied the change.

Reply:
Understood. I will take note of this.